### PR TITLE
fixed:Remove header transition effect

### PR DESCRIPTION
**Pull request by priyamourya**
Removed the transition effect in header favicon by commenting out the meta component function.

### CODE REVIEW

File: src/seo/MetaComponent.js [hunk 1]
```diff
@@ -2,33 +2,34 @@ import React from "react";
...
-      blink();
-    } else {
-      document.title = title;
-      clearInterval(blinkEvent);
-    }
-  });
+//     if (!isPageActive) {
+//       blink();
+//     } else {
+//       document.title = title;
```
Comment (smaranjitghose) [inline src/seo/MetaComponent.js:17]: Do not comment the unused code. Please remove it directly

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/seo/MetaComponent.js (hunk 1 above)
Comment (priyamourya) [reply src/seo/MetaComponent.js:17]: Alright

Comment (priyamourya) [reply src/seo/MetaComponent.js]: I did changes ,and removed extra lines of code.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
instead of removing it completely. Instead when the user is not active on the tab just display come back on that.

**Comment by smaranjitghose** [comment]
> instead of removing it completely. Instead when the user is not active on the tab just display come back on that.

Cool. We'll go ahead with it

**Comment by priyamourya** [comment]
Made changes such that when the user is not active on the tab just display come back message. #822

**Comment by smaranjitghose** [comment]
https://user-images.githubusercontent.com/46641503/113473286-64a51100-9486-11eb-8737-3e49e450c0b8.mp4

**Pull request closed without merging**